rollouts/controllers/rollout_controller.go

```diff
@@ -389,6 +439,17 @@ func isRRSSynced(rss *gitopsv1alpha1.RemoteRootSync) bool {
 	return false
 }
 
+func isRRSErrored(rss *gitopsv1alpha1.RemoteRootSync) bool {
+	if rss.Generation != rss.Status.ObservedGeneration {
+		return false
+	}
+
+	if rss.Status.SyncStatus == "Error" {
+		return true
+	}
+	return false
+}
+
```

Comment:
This is looking fantastic and very nicely done.

I am thinking we should add an overall rollout summary that answers the following:

```
Is any rollout in progress ?

If in progress:
 - Total targets to be done
 - Number of targets pending
 - Number of targets done
 - overall percentage progress.
```

What do you think ?

Reply:
Agree there should be a status, but still going back and forth how it should look. I'll do this as a follow-up.